NoteworthyUserManual.md

```diff
@@ -0,0 +1,46 @@
+# Description 
+Welcome to Noteworthy! We are the #1 Notetaking Application For Markdown files. Some of our current available functions >>> include writing, viewing, searching, listing, deleting, clearing, and converting to PDF. If you have any questions relating to Markdown the link below provides useful information
+[Markdown Guide](https://www.markdownguide.org/)
+
+# Set Up/Installation 
+You can access the required code for the Noteworthy application using the link at the bottom of this section. Furthermore, it is worth noting that this is a command line application so no external downloads are required. Finally, you need to ensure you have a "Notes" subfolder inside of your Noteworthy directory.
+[Noteworthy Code](https://github.com/augustus-thomas/noteworthy)
+
+# Current Functions
+### Home Page 
+The purpose of the homepage is to allow you to access all the available functions for the application. For example, if you want to add notes to a file then you would type "add" while on the homepage. Furthermore, if you wish to exist the app this can also be done through the home page. 
+
+### Write to file 
+This function allows you to both write to existing files and create new files to write to. This writing function can be accessed in the home page by typing "add" when prompted to specify the action you want to perform. The creation of a new file can be done by typing in the name of the file you wish to create when prompted with typing in the name of the file you wish to write to.  Furthermore, when you access this function you will be prompted to specify the name of a file you wish to write to, the date (yyyy/mm/dd), and finally what you wish to write. You can press your return key to start a new line and ctrl + d to stop writing to a file. 
```

Comment:
"with typing in the name of the file you wish to write to." is redundant and already implied in previous clause

Reply:
```suggestion
This function allows you to both write to existing files and create new files to write to. This writing function can be accessed in the home page by typing "add" when prompted to specify the action you want to perform. The creation of a new file can be done by typing in the name of the file you wish to create.  Furthermore, when you access this function you will be prompted to specify the name of a file you wish to write to, the date (yyyy/mm/dd), and finally what you wish to write. You can press your return key to start a new line and ctrl + d to stop writing to a file. 
```